shell/platform/embedder/tests/embedder_unittests.cc

```diff
@@ -2894,6 +2894,9 @@ TEST_F(EmbedderTest, CanQueryDartAOTMode) {
 }
 
 TEST_F(EmbedderTest, VerifyB143464703WithSoftwareBackend) {
+#if !defined(OS_LINUX)
```

Comment:
We ensure all tests here run on all platforms. I'd rather have this test disabled (and file a bug with a linked TODO) than skip it this way.

Reply:
Golden tests are difficult to get right for all platforms.  We could just skip it outright and re-enable it if we can get Skia gold for engine unit tests, but we'd then lose coverage even on Linux...

Reply:
(The actual differences are minor pixel differences in the gradients)

Reply:
> Golden tests are difficult to get right for all platforms.

Sigh. Very true. And, looking at this test, I am not sure why the assertions in the next presentation callback are not sufficient. The pixel test here seems to be redundant. 

It's you call on if you want to skip or disable. But please file a bug with a linked TODO either way.

Reply:
Added a TODO and filed https://github.com/flutter/flutter/issues/53784